include/mgba/internal/gba/gba.h

```diff
@@ -123,6 +124,8 @@ struct GBA {
 	bool debug;
 	char debugString[0x100];
 	GBADebugFlags debugFlags;
+
+	struct GBAHardwareExtensions hwExtensions;
```

Comment:
Would probably group this with the other components closer to the beginning.

Reply:
Fixed. Moved it under the SIO.